src/github/pullRequestManager.ts

```diff
@@ -959,15 +952,47 @@ export class PullRequestManager implements IPullRequestManager {
 				const author = matchingCommits[0].author;
 				// There is not necessarily a GitHub account associated with the commit.
 				if (author !== null) {
-					commitEvent.author.avatar_url = author.avatar_url;
+					commitEvent.author.avatarUrl = author.avatar_url;
 					commitEvent.author.login = author.login;
-					commitEvent.author.html_url = author.html_url;
+					commitEvent.author.url = author.html_url;
 				}
 			}
 		}
 	}
 
-	private async parseTimelineEvents(pullRequest: IPullRequestModel, remote: Remote, events: any[]): Promise<TimelineEvent[]> {
+	private async parseGraphQLTimelineEvents(events: any[]): Promise<TimelineEvent[]> {

```

Comment:
should these parse methods also be in the utils file?

Reply:
done